core/modules/layout/layout.entity.admin.inc

```diff
@@ -0,0 +1,425 @@
+<?php
+/**
+ * @file
+ * Provides a user interface for managing layouts for entity bundles.
+ */
+
+/**
+ * Render the settings form for listing entity layouts.
+ *
+ * @ingroup forms
+ */
+function layout_entity_admin_form($form, &$form_state, $entity_type, $bundle_arg) {
+  form_load_include($form_state, 'inc', 'layout', 'layout.admin');
+  form_load_include($form_state, 'inc', 'layout', 'layout.context.admin');
+  $form['#attached']['css'][] = backdrop_get_path('module', 'layout') . '/css/layout.admin.css';
+
+  $form['messages'] = array(
+    '#theme' => 'status_messages',
+    '#messages' => '',
+    '#weight' => -100,
+    // Prefix/suffix used to identify in AJAX requests.
+    '#prefix' => '<div id="layout-messages">',
+    '#suffix' => '</div>',
+  );
+  $bundle = field_extract_bundle($entity_type, $bundle_arg);
+  $layouts = layout_get_entity_layouts($entity_type, $bundle);
+  $entity_info = entity_get_info($entity_type);
+  if ($entity_info['label'] == 'Node') {
+    $entity_info['label'] = 'Content';
+  }
+
+  $header = array(
+    array('data' => t('Layout'), 'class' => array(
+      'layout-title',
+    )),
+    array('data' => t('Template'), 'class' => array(
+      'layout-template',
+      'priority-low',
+    )),
+    array('data' => t('Visibility conditions'), 'class' => array(
+      'layout-conditions',
+      'priority-low',
+    )),
+    array('data' => t('Operations'), 'class' => array(
+      'layout-operations',
+    )),
+  );
+
+  $rows = array();
+  foreach ($layouts['all'] as $layout) {
+    $operations = array(
+      '#type' => 'dropbutton',
+      '#links' => _layout_get_operations($layout),
+    );
+    foreach ($operations['#links'] as $link_key => $op) {
+      $operations['#links'][$link_key]['query'] = backdrop_get_destination();
+    }
+
+    // Create a link to the settings page.
+    $info = layout_get_layout_template_info($layout->layout_template);
+    $template = l($info['title'], 'admin/structure/layouts/manage/' . $layout->name . '/configure');
+
+    $row = array();
+    $row[] = theme('layout_info', array('layout' => $layout));
+    $row[] = $template;
+    $row[] = theme('layout_condition_info', array('layout' => $layout));
+    $row[] = backdrop_render($operations);
+    $class = array('layout-row');
+    if ($layout->disabled) {
+      $class[] = 'disabled';
+    }
+
+    $rows[] = array('data' => $row, 'class' => $class);
+  }
+
+  $form['stand_alone_table'] = array(
+    '#theme' => 'table',
+    '#header' => $header,
+    '#rows' => $rows,
+    '#attributes' => array('class' => array('layout-list')),
+    '#empty' => t('No layout overrides have been created for !entity_type types yet. All !entity_type types will display using the <a href="@default_layout_href">Default layout</a>.', array(
```

Comment:
This should show the bundle name (if available) rather than the entity type name:

![image](https://github.com/user-attachments/assets/6614d61f-0adf-4113-af59-790b3ec0c3f8)

In the example above, other content types have layouts applied, but not "Page". So the message should say something more like "No layout overrides have been created for this content type. Page content will display using the Default layout".

Reply:
Fixed